src/components/Feed.js

```diff
@@ -5,62 +5,136 @@ import { axiosInstance } from '../util/config';
 import { UserContext } from '../util/context';
 
 export default function Feed() {
-    const [projectData, setProjectData] = useState([]);
-    const [updateInput, setUpdateInput] = useState({ subject: '', body: '', project: '' })
-    const username = JSON.parse(window.localStorage.getItem('username'));
+  const [projectData, setProjectData] = useState([]);
+  const [updateInput, setUpdateInput] = useState({
+    subject: '',
+    body: '',
+    project: '',
+  });
+  const username = JSON.parse(window.localStorage.getItem('username'));
 
-    let { authorization } = useContext(UserContext);
-    if (!authorization) {
-        authorization = JSON.parse(window.localStorage.getItem('token'));
-    }
+  let { authorization, setUpdatesList, updatesList } = useContext(UserContext);
+  if (!authorization) {
+    authorization = JSON.parse(window.localStorage.getItem('token'));
+  }
 
-    const handleInputChange = ({ target: { name, value } }) => {
-        setUpdateInput({
-            ...updateInput,
-            [name]: value,
-        })
-    }
+  const handleInputChange = ({ target: { name, value } }) => {
+    setUpdateInput({
+      ...updateInput,
+      [name]: value,
+    });
+  };
 
-    const handleAddUpdate = () => {
-        console.log('meow');
-        axiosInstance.post(`/projects/${updateInput.project}/updates`, { subject: updateInput.subject, body: updateInput.body }, { headers: { 'authorization': 'Bearer ' + authorization } }).then((res) => {
-            console.log('res-post:', res);
-        })
-    }
+  const handleAddUpdate = () => {
```

Comment:
Could have been renamed to `createUpdate` because `handle` is usually associated with events